recipes/mender-app-update-module/debian-master/rules

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	true
+
+override_dh_auto_install:
+	DESTDIR=debian/mender-app-update-module make install
+
+override_dh_auto_test:
+	true
+
+override_dh_strip:
+	true
```

Comment:
Also these two should be able to run with defaults, why do you need to override?

Reply:
fixed